cpp/include/ray/api/actor_creator.h

```diff
@@ -71,6 +72,11 @@ class ActorCreator {
     return *this;
   }
 
+  ActorCreator &SetRuntimeEnv(ray::RuntimeEnv runtime_env) {
```

Comment:
It's better to use `const ray::RuntimeEnv &runtime_env` to instead of `ray::RuntimeEnv runtime_env` here.